source/table.cpp

```diff
@@ -47,6 +48,34 @@ spv_context spvContextCreate(spv_target_env env) {
 
 void spvContextDestroy(spv_context context) { delete context; }
 
+spv_validator_options spvValidatorOptionsCreate() {
+  return new spv_validator_options_t;
+}
+
+void spvValidatorOptionsDestroy(spv_validator_options options) {
+  delete options;
+}
+
+void spvValidatorOptionsSetMaxStructMembers(spv_validator_options options,
+                                            const char* limit) {
```

Comment:
Should take an integer argument directly.  Don't parse a string in this API.
(And this method should be enum-driven. See above.)